### PR TITLE
Add hi-IN.md

### PR DESCRIPTION
**Pull request by loungeclown**
Here's my contribution to the modpack! Includes all the phrases translated by me and confirmed by my parents and google translate.

### CODE REVIEW

**Comment by Madis0** [comment]
Conflicts with #434. We have to find an agreement on which one is better or to find best strings from both versions and combine them to one.

File: Description/hi-IN.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+> **प्रश्न**: क्या मुझे सर्वर पर मॉडपैक की आवश्यकता है?
+> 
+> **उत्तर**: नहीं, यह उन सभी सर्वरों पर काम करता है जो वैनिला क्लाइंट को अनुमति देते हैं। [विस्तृत उत्तर और सर्वर युक्तियाँ यहाँ हैं।][13] 
+> 
+> [![](https://i.ibb.co/gr9mSxW/image.png) यदि आपको एक होस्ट की आवश्यकता है, तो यहां क्लिक करें - पहले महीने के लिए 25% छूट प्राप्त करने के लिए!][14]
+
+
+> **प्रश्न**: मेरे पास और प्रश्न हैं!
+> 
+> **उत्तर**: [आओ और हमारे साथ डिस्कोर्ड पर चैट करें!][10]
```
Comment (Probattler) [inline Description/hi-IN.md:70]: Cut आओ 
Add आइये 
It's more appropriate

**Comment by Probattler** [comment]
Its better to keep "Fabric, Forge, OptiFine (yes let F be capital) and perhaps Minecraft too" To allow better readability and understandability.

**Comment by loungeclown** [comment]
> Its better to keep "Fabric, Forge, OptiFine (yes let F be capital) and perhaps Minecraft too" To allow better readability and understandability.

ehh i am kind of out of this translation thing. (also who would use minecraft in hindi ¯\_(ツ)_/¯) and i mean... dude who wants to play minecraft would already know english... i mean considering literacy upgrowth in india, i think translating this in hindi is a timewaste. but if you are doing it, no problem! i just say, good luck.

**Pull request closed without merging**